### ISSUE TITLE
Support custom loss function for DNNRegressor

### ISSUE THREAD

**Issue by Marcnuth** [tensorflow/tensorflow#9586]
DNNRegressor is commonly used, and it does not support customize the loss function, which limits itself.

I wanna know if there are any schedule to support the customize loss function for DNNRegressor or is there any difficulties for supporting that?

**Comment by martinwicke** [comment]
What other loss function would you like to have?

**Comment by ispirmustafa** [comment]
You can use `tf.contrib.learn.DNNEstimator` which takes a `head` as argument.
You can define any loss fn you want via `head`.

**Comment by Marcnuth** [comment]
I will use DNNEstimator, thanks for replying.
